docker/Dockerfile

```diff
@@ -55,7 +56,6 @@ RUN apk --no-cache add gcc \
   libusb-dev \
   gpsd-dev \
   python3-dev \
-  boost-dev \
```

Comment:
I think the boost library is still needed for the Bustache extension if that is build with the image

Reply:
you're right, I missed that -- fixed in 0732b67f91282dc048b9be357456f166c733e053